addons/web/static/src/views/fields/field.js

```diff
@@ -162,33 +162,39 @@ export class Field extends Component {
 Field.template = "web.Field";
 
 Field.parseFieldNode = function (node, models, modelName, viewType, jsClass) {
-    const name = node.getAttribute("name");
```

Comment:
you could keep this line s.t. you can default string and field keys like the others